pai-management/bootstrap/drivers/drivers.yaml.template

```diff
@@ -29,9 +29,7 @@ spec:
         app: drivers-one-shot
     spec:
       hostNetwork: true
-      hostPID: true
-      nodeSelector:
-        machinetype: gpu
+      hostPID: true      
```

Comment:
After this change, we will unable to remove or stop drivers from a specific node. I suggest to use ```NotIn``` here. And you can randomly chose a tag such as ```stop-drivers=true```. This label won't be tagged into the node by default. If admin wanna remove this component from the node, he or she could manually add this tag.

Reply:
Ok, and makes node-exporter the same way.